index.html

```diff
@@ -57,9 +57,11 @@ <h3>Backgrounds</h3>
         <img class="hide bowtie accessories accessories-js" src="bab-assets/bowtie.png" alt="bowtie">
         <img class="hide smartwatch accessories accessories-js" src="bab-assets/smartwatch.png" alt="smartwatch">
         <img class="hide sunglasses accessories accessories-js" src="bab-assets/sunglasses.png" alt="sunglasses">
+        <img class="hide backgrounds backgrounds-js" src="bab-assets/blue.png" alt="blue">
         <img class="hide backgrounds backgrounds-js" src="bab-assets/park.png" alt="park">
         <img class="hide backgrounds backgrounds-js" src="bab-assets/beach.png" alt="beach">
         <img class="hide backgrounds backgrounds-js" src="bab-assets/outerspace.png" alt="outerspace">
+        <img class="hide backgrounds backgrounds-js" src="bab-assets/yellow.png" alt="yellow">
```

Comment:
Had a conversation with Kyle on these two things to figure out what they were for.  We are replacing the backgrounds for the ones labeled blue and yellow with the img's added here.